### ISSUE TITLE
Need Pip Support for v3.1.1.1

### ISSUE THREAD

**Issue by fyffep** [openbabel/openbabel#2571]
I am unable to install Openbabel's Python module. I believe that Openbabel needs to build the wheel for Python 3.9 and 3.10 with their latest version for pypi.

## Environment Information
Open Babel version: Latest (3.1.1.1)
Operating system and version: x64-bit Windows 10

## Expected Behavior
I want to install Openbabel from a `requirements.txt` file, so I ran `pip install openbabel`.

## Actual Behavior
```
>pip install openbabel
Collecting openbabel
  Using cached openbabel-3.1.1.1.tar.gz (82 kB)
  Preparing metadata (setup.py) ... done
Installing collected packages: openbabel
  DEPRECATION: openbabel is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
  Running setup.py install for openbabel ... error
  error: subprocess-exited-with-error

  × Running setup.py install for openbabel did not run successfully.
  │ exit code: 1
  ╰─> [16 lines of output]
      running install
      C:\[Path]\Python\Python310\lib\site-packages\setuptools\command\install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build_ext
      Warning: invalid version number '3.1.1.1'.
      Guessing Open Babel location:
      - include_dirs: ['C:\\[Path]\\Python\\Python310\\include', 'C:\\[Path]\\Python\\Python310\\Include', '/usr/local/include/openbabel3']
      - library_dirs: ['C:\\[Path]\\Python\\Python310\\libs', 'C:\\[Path]\\Python\\Python310', 'C:\\[Path]\\Python\\Python310\\PCbuild\\amd64', '/usr/local/lib']
      building 'openbabel._openbabel' extension
      swigging openbabel\openbabel-python.i to openbabel\openbabel-python_wrap.cpp
      swig.exe -python -c++ -small -O -templatereduce -naturalvar -IC:\[Path]\Python\Python310\include -IC:\[Path]\Python\Python310\Include -I/usr/local/include/openbabel3 -o openbabel\openbabel-python_wrap.cpp openbabel\openbabel-python.i

      Error: SWIG failed. Is Open Babel installed?
      You may need to manually specify the location of Open Babel include and library directories. For example:
        python setup.py build_ext -I/usr/local/include/openbabel3 -L/usr/local/lib
        python setup.py install
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> openbabel

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.

[notice] A new release of pip is available: 23.0 -> 23.0.1
[notice] To update, run: python.exe -m pip install --upgrade pip
```

**Comment by JoaoSerina** [comment]
I am also experiencing this issue. 

I have also tried installing python bindings on python 3.6 and 3.9 and will still have error. 

I even tried using the "--use-pep517" option and still get the error.

**Comment by PatReis** [comment]
Am not sure that latest `openbabel` version can be installed with pip only. See this issues: #2403 #2271
But I am very much looking forward to this feature. Conda install should work.

**Comment by Garyfallidis** [comment]
Same issue here!

**Comment by Garyfallidis** [comment]
Updated pip installers would be great! We know it is a lot of work. But hang in there Openbabel team!

**Comment by andizuend** [comment]
I agree, updated pip installers that work on Windows would be great. 
Here is what worked for me (for openbabel 3.1.1, Python 3.10.10):
**Openbabel installation**
- download & install openbabel 3.1.1 64-bit version for Windows (download the installer file from https://github.com/openbabel/openbabel/releases/ )
- IMPORTANT step after if it should be used via Python openbabel / pybel later: delete BABEL_DATADIR variable in the environment variables of Windows; (type in search box: "edit environment variables" and open the control panel for that.

**Python bindings to openbabel**
- to install Python openbabel bindings on Windows, I only succeeded when using a precompiled .whl wheel downloaded from [https://www.lfd.uci.edu/~gohlke/pythonlibs/#openbabel](https://www.lfd.uci.edu/~gohlke/pythonlibs/#openbabel); for reasons see this [related post](https://stackoverflow.com/questions/42151301/how-do-i-install-openbabel-for-python-3-6-in-windows-10).
- Choose an amd64 version that is compatible with the current Python version (check for compatible versions in command prompt via: ` pip debug --verbose` ).
- Then change the directory in the cmd prompt (run as admin) to your download directory and install using a pip command like:  `pip install openbabel-3.1.1-cp310-cp310-win_amd64.whl`

**Comment by ulricharmel** [comment]
Hi please can anyone guide me on how to pip install openbabel in virtual environment on ubuntu system. I keep getting the same error mentioned in this issue. I am trying to do this with a python 3.10 virtual environment on ubuntu 22.04

**Comment by nbehrnd** [comment]
An alternative approach is to use openbabel-wheel which equally is available
on pypi and hence can be used with pip.  Commands and functionality provided
by this bypass are the same, so keep your line of `from openbabel import pybel`
and other instructions in the Python scripts.

How to proceed:

- if the virtual environment is pristine, run `pip install openbabel-wheel`
- if it is not because you once run a `pip install openbabel`, first remove
  the old libraries by `pip uninstall openbabel` and subsequently run `pip
  install openbabel-wheel` (or just remove the virtual environment for good
  and create a new venv).
- if the virtual environment uses a `requirements.txt` file, substitute the
  entry `openbabel` by `openbabel-wheel`.  Then, as usual, run `pip install
  -r ./requirements.txt`.
